### PR TITLE
Add ability to SELECT Max (field)

### PR DESCRIPTION
**Pull request by MarcusFulbright**
I've been using Aura.SqlQuery at work for the past two weeks. We have to do some `Select Max (field) From <table>` queries. It'd be nice to use this library for those queries as well.

### CODE REVIEW

File: src/Common/SelectInterface.php [hunk 1]
```diff
@@ -232,4 +232,15 @@ public function union();
...
+    /**
+     *
+     * Will select max value from a specified column
+     *
+     * @param string $col name of the Column
+     *
+     * @return self
+     *
+     */
+    public function max($col);
```
Comment (staabm) [inline src/Common/SelectInterface.php:245]: This breaks BC

Comment (MarcusFulbright) [inline src/Common/SelectInterface.php:245]: The functionality was implemented in Common/Select. All select classes inherit from that correct? Where is the BC break?

**Comment by pmjones** [comment]
/me thinks

Have we tried `$select->from('table')->cols(['MAX(foo) AS max_foo'])` ?

**Comment by MarcusFulbright** [comment]
@pmjones that passes the same test in this pull request (just requires a slight whitespace change). Somehow I missed the `count()` example in the docs too. Totally my bad.

Now that I know functions can be in the `$cols` array, implementing `$select->max('col')` feels like a slippery slope that leads to implementing all the functions. 

Good thing this pull request only took a few minuets to code.

**Pull request closed without merging**